ci/infra/testrunner/skuba/skuba.py

```diff
@@ -132,8 +138,48 @@ def _wait_node_joined(self, role, node, timeout=60, backoff=10):
                 raise Exception((f'Node {node_name} not shown ready after {timeout} seconds'
                                  f'{". Last error:"+str(last_error) if last_error else ""}'))
             time.sleep(backoff)
+
+
+    def _wait_apiserver_ready(self, node, timeout=60, backoff=10):
+        apiserver_healthz = 'curl -Ls --insecure https://localhost:6443/healthz'
+
+        try:
+            self._wait_condition("master", node, apiserver_healthz, 'ok', timeout=timeout)
+        except AssertionError as ex:
+            node_name = self.platform.get_nodes_names("master")[node]
+            raise Exception(f'Error waiting apiserver at {node_name} to become ready: {ex}')
```

Comment:
pls use `from ex` to use explicit exception chaining[0] if you are raising a different error other than the original one


```suggestion
            raise Exception(f'Error waiting apiserver at {node_name} to become ready') from ex
```

[0] https://www.python.org/dev/peps/pep-3134/#explicit-exception-chaining